bika/lims/browser/client/views/analysisspecs.py

```diff
@@ -5,77 +5,24 @@
 # Copyright 2018 by it's authors.
 # Some rights reserved. See LICENSE.rst, CONTRIBUTORS.rst.
 
+from bika.lims import api
 from bika.lims import bikaMessageFactory as _
-from bika.lims.browser import BrowserView
-from bika.lims.browser.bika_listing import BikaListingView
-from bika.lims.permissions import *
-from bika.lims.utils import isActive
-from bika.lims.utils import tmpID
-from plone.app.layout.globals.interfaces import IViewView
-from Products.CMFCore.utils import getToolByName
-from Products.CMFPlone.utils import _createObjectByType
-from zope.interface import implements
+from bika.lims.controlpanel.bika_analysisspecs import AnalysisSpecsView
+from bika.lims.permissions import AddAnalysisSpec
 
 
-class ClientAnalysisSpecsView(BikaListingView):
-    implements(IViewView)
+class ClientAnalysisSpecsView(AnalysisSpecsView):
 
     def __init__(self, context, request):
         super(ClientAnalysisSpecsView, self).__init__(context, request)
-        self.catalog = 'bika_setup_catalog'
-        self.contentFilter = {
-            'portal_type': 'AnalysisSpec',
-            'sort_on': 'sortable_title',
-            'getClientUID': context.UID(),
-            'path': {
-                "query": "/".join(context.getPhysicalPath()),
-                "level": 0
-            }
-        }
-        self.context_actions = {}
+        self.contentFilter["getClientUID"] = api.get_uid(context)
 
-        self.show_sort_column = False
-        self.show_select_row = False
-        self.show_select_column = True
-        self.pagesize = 50
-        self.form_id = "analysisspecs"
+    def before_render(self):
+        """Before template render hook
+        """
+        # We want to display the nav tabs, so we do NOT want disable_border in
+        # the request. Thus, do not call super.before_render
```

Comment:
Note that `request.set('disable_border', False)` will not work because `ploneview` only checks if `disable_border` exists in the request, but not its value: https://github.com/plone/Products.CMFPlone/blob/master/Products/CMFPlone/browser/ploneview.py#L49